### PR TITLE
add submission layout

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: client/uikit/AppBar/index.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const UserBadge = ({ firstName = '', lastName = '', title }) => {
 };
 UserBadge.propTypes = {
   firstName: PropTypes.string.isRequired,
-  lastirstName: PropTypes.string.isRequired,
```
Comment (hlminh2000) [inline client/uikit/AppBar/index.js]: me derped... thanks!

**Comment by ciaranschutte** [comment]
> The Story source and PropTypes could be cleaner

Where is this? I didn't add a story...

**Comment by ciaranschutte** [comment]
@hlminh2000 @d8660091  Can I go ahead and merge this? @d8660091  not sure what you mean about Story

**Comment by d8660091** [comment]
@ciaranschutte I was looking at a wrong story, nvm

**Pull request merged**